docs/Sources/tuist/tuist.docc/Articles/Tuist/Extensions/generation-environment.md

```diff
@@ -0,0 +1,49 @@
+# Generation-time configuration
```

Comment:
It feels a bit odd having this page under the "extensions" section, but I can't think of a different place for it. The content has to do with "things that you can do from your manifests", so it feels closer to the concept of "project description helpers" than "tuist extensibility". What if we have a page that touches on "manifest features" / "manifest capabilities"